### PR TITLE
Add pagination feature

### PR DESCRIPTION
**Pull request by hkedia321**
Added pagination feature!
image

### CODE REVIEW

File: src/components/VisitorForm.vue [hunk 1]
```diff
@@ -92,7 +92,7 @@
                 return this.visitor.address === ''
             },
             invalidPhone() {
-                return this.visitor.phone === '' || !this.visitor.phone.match(/^\d{9,15}$/)
+                return false//this.visitor.phone === '' || !this.visitor.phone.match(/^\d{9,15}$/)
```
Comment (pararang) [inline src/components/VisitorForm.vue]: why you disabled this validation, @hkedia321 ?

Comment (hkedia321) [reply src/components/VisitorForm.vue]: @pararang had it for testing. Now removed it! You can merge now

**Comment by hkedia321** [comment]
@pararang can you merge this?

**Comment by pararang** [comment]
merged, @hkedia321 .

but still not set the latest record as the first. please set the order by `visit_at` on ascending.

**Comment by kgoel085** [comment]
@pararang Hi i have done the pagination work and please review and merge. Thanks. Here is a little preview of what i have done 
image

**Comment by geek484** [comment]
Hi @pararang, @kgoel085's implementation looks good. You may revert this PR and merge his!

**Comment by kgoel085** [comment]
Hi, I just made a new pull request. Can u review it too. In addition. i
have also made a validate phone implementation when adding visitor

**Pull request merged**